python/JetMaker_cfi.py

```diff
@@ -49,7 +49,7 @@
       "Spring16_25nsV6_MC_L3Absolute_AK8PFchs.txt"), 
     jecShift                  = cms.double(0), 
     jerShift                  = cms.int32(1), 
-    jetPtMin                  = cms.double(300),
+    jetPtMin                  = cms.double(500),
```

Comment:
@jking79 @EJDomi this should not be changed here. As we discussed you should change the variable leadingJetPtMin in the VLQAna_cfi

In general, never change the cfi files under the python dir.